spec/fixtures/_complete_rizzo.json

```diff
@@ -0,0 +1,280 @@
+    "box_download_checksum": "37f67caf1038992207555513504e37258c29e2e9",
+    "box_download_checksum_type": "sha1",
+    "memory": "1024",
+    "netmask": "255.255.255.0",
+    "update_packages": true,
+    "update_packages_command": "yum -y update",
+    "shutdown": true,
+    "shutdown_command": "/sbin/shutdown -h now"
+  },
+  "control_repos": [
+    "/Users/jeff/projects/acme/puppetdata",
+    "/Users/jeff/projects/acme/ghoneycutt-modules"
+  ],
+  "puppetmaster": {
+    "name": [
+      "infra-puppetfourca",
+      "infra-puppetfour"
+    ],
+    "modulepath": [
+      "./modules",
+      "./puppetdata/modules",
+      "./ghoneycutt/modules"
+    ],
+    "synced_folders": {
+      "/repos/puppetdata": {
+        "local": "/Users/jeff/projects/acme/puppetdata",
+        "owner": "root",
+        "group": "root"
+      },
+      "/repos/ghoneycutt": {
+        "local": "/Users/jeff/projects/acme/ghoneycutt-modules",
+        "owner": "root",
+        "group": "root"
+      }
+    }
+  },
+  "nodes": [
+    {
+      "name": "infra-puppetfourca",
+      "hostname": "infra-puppetfourca01.dev.acmehq.acmeint.com",
+      "forwarded_ports": [
+        {
+          "guest": "8140",
+          "host": "8140"
+        }
+      ],
+      "ip": "172.16.100.5",
+      "memory": "2048"
+    },
+    {
+      "name": "infra-puppetfour",
+      "hostname": "infra-puppetfour01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.6",
+      "memory": "2048"
+    },
+    {
+      "name": "infra-test",
+      "hostname": "infra-test01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.7"
+    },
+    {
+      "name": "infra-dnsmasterext",
+      "hostname": "infra-dnsmasterext01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.8"
+    },
+    {
+      "name": "infra-dnsmasterint",
+      "hostname": "infra-dnsmasterint01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.9"
+    },
+    {
+      "name": "infra-dnsslaveext",
+      "hostname": "infra-dnsslaveext01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.10"
+    },
+    {
+      "name": "infra-dnsslaveint",
+      "hostname": "infra-dnsslaveint01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.11"
+    },
+    {
+      "name": "infra-logagg",
+      "hostname": "infra-logagg01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.12"
+    },
+    {
+      "name": "infra-mx",
+      "hostname": "infra-mx01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.13"
+    },
+    {
+      "name": "infra-provisioner-fe",
+      "hostname": "infra-provisioner-fe01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.14"
+    },
+    {
+      "name": "infra-provisioner-scheduler",
+      "hostname": "infra-provisioner-scheduler01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.15"
+    },
+    {
+      "name": "infra-provisioner-worker",
+      "hostname": "infra-provisioner-worker01.dev.acmehq.acmeint.com",
+      "ip": "172.16.100.16"
+    },
+    {
+      "name": "axpetl-app",
```

Comment:
all of these names seem a little too close to home. Let's just pick a few nodes with names like puppet, web, app and db with hostnames like puppet.dev.example.com